app/views/movies/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1><%= @movie.title %></h1>
+<p>Creation Year: <%= @movie.creation_year %></p>
+<p>Genre: <%= @movie.genre %></p>
+
+<div id="movie_actors">
+  <h2>Actors</h2>
+  <% @movie.youngest_to_oldest_actors.each do |actor| %>
+    <p><%= actor.name %></p>
+  <% end %>
+
+  <h3>Average Actor Age: <%= @movie.average_actor_age %></h3>
+
+  <div id="add_movie_actor">
+    <%= form_with url: "/movies/#{@movie.id}/actors", method: :post, data: { turbo: false }, local: true do |f| %>
```

Comment:
Nice path & method setup, and form overall!

Reply:
You could also use a path like `"/actor_movies"`, and find a way to append the movie_id to the sent data... I can think of two ways to do that. 🤔